### PR TITLE
FFM-7066: DO NOT MERGE (awaiting GA) All environments page

### PR DESCRIPTION
**Pull request by StrawberryPad**
Includes a move+rename of the existing topic: Filter Flags by State >> View and Filter Flags
The environments page is not released yet, waiting for final code changes (ETA week of 17April)
[**Preview link**](https://643d25ce62ad650f962146b1--harness-developer.netlify.app/docs/feature-flags/ff-using-flags/filtering-flags-by-state#view-flag-states-in-different-environments)

### CODE REVIEW

**Review by Kira-Jennings — DISMISSED** [review]
Looks good!

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md [hunk 1]
```diff
@@ -1,43 +1,60 @@
 ---
-title: Filter Flags by State
+title: View and Filter Flags
```
Comment (KimberlyFields) [inline docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:2]: Use sentence case for titles.

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md [hunk 2]
```diff
@@ -1,43 +1,60 @@
...
 
+**To view filter your flags:**
 
+1. In Harness, go to your project, and then select **Feature Flags**.
+
+    The dashboard displays the total number of flags, and the number of flags in each state. For details, go to [Flag states](#flag-states) below. 
+
+    ![A screenshot of the Permanent Flags tile selected to filter for permanent flags.](../static/9-filtering-flags-by-state-01.png)
+
+3. Select **All Flags**, or use one of the available filters, for example, **Permanent Flags**.
```
Comment (KimberlyFields) [inline docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:23]: This should be step 2.

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md [hunk 3]
```diff
@@ -1,43 +1,60 @@
...
-| Active Flags | Flags that have been evaluated in the last 7 days |
-| Potentially Stale Flags | Flags are marked as potentially stale if in the past 60 days they haven't been changed or evaluated,their default rules or target rules haven’t been added to or updated, they haven’t been toggled on or off.|
-
-:::caution
- A Flag can be marked as both Active and Potentially Stale if it has been Evaluated via an SDK but no other changes have been made in over 60 days.  For example, if you Evaluated `Flag_A` yesterday using an SDK, but haven’t made any changes on the Harness Platform in over three months, the Flag will be marked as Active and Potentially Stale on the Platform. 
+| --- | --- |
+| **Enabled Flags** | Flags that are currently toggled on. |
+| **Permanent Flags** | Flags you intend to stay in your systems indefinitely and that you marked as permanent when creating them. Permanent flags are never marked as stale. |
+| **Recently Changed Flags** | Flags that have been changed in the last 24 hours. Changes include enabling or disabling a flag, or adding new Rules or Targets. |
+| **Active Flags** | Flags that have been evaluated in the last 7 days |
```
Comment (KimberlyFields) [inline docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:35]:
```suggestion
| **Active Flags** | Flags that have been evaluated in the last 7 days. |
```

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md [hunk 4]
```diff
@@ -1,43 +1,60 @@
...
+| **Potentially Stale Flags** | Flags are marked as potentially stale if in the past 60 days they haven't been changed or evaluated,their default rules or target rules haven’t been added to or updated, they haven’t been toggled on or off.|
```
Comment (KimberlyFields) [inline docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:36]:
```suggestion
| **Potentially Stale Flags** | Flags are marked as potentially stale if in the past 60 days they haven't been changed or evaluated, their default rules or target rules haven’t been added to or updated, or they haven’t been toggled on or off.|
```

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md [hunk 5]
```diff
@@ -1,43 +1,60 @@
...
+
+:::info note
+ A flag can be marked as both **Active** and **Potentially Stale** if it has been evaluated via an SDK but no other changes have been made in over 60 days. For example, if you evaluated `Flag_A` yesterday using an SDK, but haven’t made any changes on the Harness Platform in over three months, the flag will be marked as Active and Potentially Stale on the Platform.
```
Comment (KimberlyFields) [inline docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:39]: You used bold on Active and Potentially Stale in the first sentence, but not in the second sentence. 

Also, the last part of the second sentence, "on the Platform". Use "Harness Platform".

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md [hunk 6]
```diff
@@ -1,43 +1,60 @@
...
-To view filter your Flags:
+## View flag states in different environments
+
+The Feature Flags dashboard can display the flag states and details for a selected environment, or a summary of flag states across all environments.
+
+To view flag states in different environments:
+
+1. In Harness, go to your project, and then select **Feature Flags**.
+ 
+1. In **Environments**, choose an environment from the drop-down, or **All Environments**.
```
Comment (KimberlyFields) [inline docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:51]:
```suggestion
2. In **Environments**, select a specific environment, or **All Environments**.
```

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md (hunk 1 above)
Comment (StrawberryPad) [reply docs/feature-flags/ff-using-flags/filtering-flags-by-state.md:2]: Oh wow, I've been following the title style in the docs, but it turns out ALL of the FF titles are wrong >camel case. Let me fix those in one go after I merge this: [FFM-7548](https://harness.atlassian.net/browse/FFM-7548)

File: docs/feature-flags/ff-using-flags/filtering-flags-by-state.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (StrawberryPad) [reply docs/feature-flags/ff-using-flags/filtering-flags-by-state.md]: Yup, they're capitalized.

**Review by KimberlyFields — DISMISSED** [review]

**Pull request merged**